libraries/rush-lib/src/cli/scriptActions/PhasedScriptAction.ts

```diff
@@ -660,7 +689,10 @@ export class PhasedScriptAction extends BaseScriptAction<IPhasedCommandConfig> {
       );
 
       const executeOptions: IExecutionOperationsOptions = {
-        createOperationsContext,
+        executeOperationsContext: {
+          ...initialCreateOperationsContext,
+          projectChangeAnalyzer: initialState
+        },
```

Comment:
```suggestion
        executeOperationsContext: createOperationsContext
```

Why are you overriding `projectChangeAnalyzer` here?

Reply:
It's removed from `createOperationsContext` in this PR.